0004/README.md

```diff
@@ -0,0 +1,416 @@
+1. Create a PR for the repository containing
+    1. Name of the build platform
+    1. Build ID
+    1. Desired SLSA levels
+    1. List of URIs that resolve to proof of SLSA conformance
+    1. List of URIs that resolve to materials for verifying provenance
+    1. List of URIs that resolve to the build platform's public
+        vulnerability disclosures
+
+1. Complete the terms and conditions form.
+1. Send the PR for review.
+
+The community will review the PR and merge it when a quorum of maintainers
+approve it. When reviewing the PR, the community will assume that the
+information in the proof of conformance is truthful, and they will approve it if
+the described build platform meets the requirements for the desired SLSA levels.
+
+Once the PR is merged and the terms are signed, the OpenSSF grants the build
+platform a limited license to use the SLSA logo on its website and other
+promotional materials. Should the build platform fail to adhere to the program's
+terms or meet its requirements, the OpenSSF will revoke that license and, if
+necessary, enforce its trademarks. 
+
+#### Tier two: Third-party audit
+
+The third-party audit tier requires that a build platform:
+
+-  Provide the auditor access to the build system and any supporting
+    materials needed for the auditor to gain trust in the build system.
+-  Accept that the audit report will be made public regardless of the
+    audit's outcome. Audit reports may be delayed up to 90 days to respond to
+    vulnerabilities in the build platform, but they must be published within 30
+    days of patching the vulnerability.
+-  Provide verifiers with any materials needed to verify provenance signed
+    by the build system. Publishing these materials on a domain registered to
+    the build system meets this requirement.
+-  Notify known build platform users of any security incidents that
+    potentially impact the integrity of artifacts or provenance generated by
+    the build system.
+
+This tier's requirements are enforced by the auditor.   
+The OpenSSF will run a community-maintained repository of approved auditors and
+audited build platforms. This repository may be the same one used for the
+self-attestation tier.  
+In order to participate in this tier, build platforms should
+
+1. Successfully complete an audit with an approved auditor
+1. Create a PR for the repository containing
+    1. Build platform name
+    1. Audited SLSA levels
+    1. URI that resolves to the public audit report
+    1. (Optional) Build ID
+    1. (Optional) List of URIs that resolve to proof of SLSA conformance
+    1. (Optional) List of URIs that resolve to materials for verifying
+        provenance
+    1. (Optional) List of URIs that resolve to the build platform's
+        public vulnerability disclosures
+
+1. Complete the terms and conditions form.
+1. Send the PR for review.
+1. Have the auditor approve the PR with a comment indicating the audited
+    SLSA levels
```

Comment:
I've changed it so that the auditor sends the PR.

There's no need for an approval from the build platform because the audit report must always be made public and requiring an approval would allow the build platform to suppress the report.